man/unattended-upgrade.8

```diff
@@ -38,8 +39,18 @@ help output
 \fB-d\fR, \fB\-\-debug\fR
 extra debug output into /var/log/unattended-upgrades.log
 .TP
+\fB--apt-debug\fR
+detailled APT/LibAPT output for debugging
```

Comment:
detailed

Reply:
This is still not fixed but i can fix that after merging.

Reply:
Fixed now, sorry for the slowness (and force pushed)